cmd/crc/cmd/setup.go

```diff
@@ -22,7 +22,7 @@ var (
 
 func init() {
 	setupCmd.Flags().Bool(crcConfig.ExperimentalFeatures, false, "Allow the use of experimental features")
-	setupCmd.Flags().StringP(crcConfig.Bundle, "b", constants.DefaultBundlePath(), "Bundle to use for VM")
+	setupCmd.Flags().StringP(crcConfig.Bundle, "b", constants.GetDefaultBundlePath(), "Bundle to use for VM")
```

Comment:
I think this can be merged with the previous commit, I expect they are changing exactly the same lines of code, and the intermediate `DefaultBundlePath()` state is not really useful to have.

Reply:
@cfergeau done.

Reply:
The commit log would deserve slightly more rewording, it's really not important to explicitly say that it was renamed and turned into a function. Just `constanst: Change DefaultBundlePath to GetDefaultBundlePath()` as the shortlog does the trick, together with a longer explanation in the long log